Annotation/constituent_golden_set.json

```diff
@@ -385,7 +404,7 @@
                     }
                 ],
                 "type": "constituent",
-                "clause": "",
+                "clause": "how many people talk with God even a fraction of the time they spend on their cell phone",

```

Comment:
Degree clause so maybe not interrogative

Reply:
Im going to leave these examples/comments for now, and we can discuss if we just want to get rid of them